artemis-core-client/src/main/java/org/apache/activemq/artemis/core/message/impl/CoreMessage.java

```diff
@@ -106,11 +106,12 @@
    private final CoreMessageObjectPools coreMessageObjectPools;
 
    public CoreMessage(final CoreMessageObjectPools coreMessageObjectPools) {
+      new Exception("new coreMessage").printStackTrace();
       this.coreMessageObjectPools = coreMessageObjectPools;
    }
 
    public CoreMessage() {
-      this.coreMessageObjectPools = null;
+      this((CoreMessageObjectPools)null);
```

Comment:
?

Reply:
this is just using the same constructor, that is doing the same thing

Reply:
I will remove it.. as it's not related to anything I needed here.